src/app/child-dev-project/notes/notes-migration/notes-migration.service.ts

```diff
@@ -0,0 +1,118 @@
+import { Injectable } from "@angular/core";
+import { EntityMapperService } from "../../../core/entity/entity-mapper.service";
+import { Note } from "../model/note";
+import { User } from "../../../core/user/user";
+
+@Injectable({
+  providedIn: "root",
+})
+export class NotesMigrationService {
+  private static updateNoteText(note: Note, additionalUsers: string[]) {
```

Comment:
I know this function does not use any class variables, but defining it as static and making it the first function of the class confuses readers quite a lot. I would keep it a class variable and place it after the first usage in line 60.

Also please remove the whitespaces between the comments and the functions in this file